README.md

```diff
@@ -68,7 +68,7 @@ local_conf_header:
 To configure the machine, you have to update the `machine` variable.
 And the same for the `distro`.
 
-For further information, you can read more at <https://kas.readthedocs.io/en/1.0/index.html>
+For further information, you can read more at <https://kas.readthedocs.io/en/latest/index.html>
```

Comment:
I'm not sure if this is a good idea or not given that we use version 8 in the kas configuration file. We should stick to whatever that configuration uses. What is the problem that you are trying to solve here?

Reply:
Hi,
Sorry for the late answer.

My thinking was more that the Kas people probably think their latest release is the greatest.
As Yocto people do :)

The 1.0 release of Kas is from 21 May 2019.  That's about the time of the Warrior release of Yocto.

It was just a quick online editing.  Not something to solve, just keep the documentation in line with the version most people will download and install first.
And a bit of promotion of Kas, too.

br

Reply:
Fair. I'm fine with that.